look/__brick__/look/look_data/specific_look_data/motion_look_data.dart

```diff
@@ -16,4 +18,33 @@ class MotionLookData {
   final Duration durationFast;
   final Duration durationNormal;
   final Duration durationSlow;
+
+  @override
+  ThemeExtension<MotionLookData> copyWith({
+    Duration? durationVeryFast,
+    Duration? durationFast,
+    Duration? durationNormal,
+    Duration? durationSlow,
+  }) {
+    return MotionLookData(
+      durationVeryFast: durationVeryFast ?? this.durationVeryFast,
+      durationFast: durationFast ?? this.durationFast,
+      durationNormal: durationNormal ?? this.durationNormal,
+      durationSlow: durationSlow ?? this.durationSlow,
+    );
+  }
+
+  @override
+  ThemeExtension<MotionLookData> lerp(ThemeExtension<MotionLookData>? other, double t) {
+    if (other is! MotionLookData) {
+      return this;
+    }
+
+    return MotionLookData(
+      durationVeryFast: other.durationVeryFast,
+      durationFast: other.durationFast,
+      durationNormal: other.durationNormal,
+      durationSlow: other.durationSlow,
```

Comment:
Can you do `return other.type as MotionLookData` rather than writing all of this?